### PR TITLE
Add Mastodon social icon

### PR DESCRIPTION
**Pull request by FestplattenSchnitzel**
- [ ] Run the tests and see them pass
- [x] Rebase your branch on top of `develop`
- [ ] Include tests for features you introduced / bugs you fixed

This PR does not fix any issues.

### CODE REVIEW

File: sipa/templates/base.html [hunk 1]
```diff
@@ -357,6 +357,13 @@ <h2>{{ _("Kontakt") }} </h2>
                                 <hr>
 
                                 <div class="row">
+                                    <div class="col-xs-4 text-center">
```
Comment (ibot3) [inline sipa/templates/base.html:360]: Needs to be changed to `col-xs-3`. This also applied for the other 3 icons.

File: sipa/templates/base.html [hunk 2]
```diff
@@ -357,6 +357,13 @@ <h2>{{ _("Kontakt") }} </h2>
...
+                                        <a class="btn btn-social-icon btn-mastodon"
+                                           href="https://social.anoxinon.de/@ag_dsn"
+                                           aria-label="Mastodon">
+                                            <span class="fa fa-mastodon" aria-hidden="true"></span>
```
Comment (ibot3) [inline sipa/templates/base.html]: Icon doesn't exist in fontawesome 4.
I upgraded our FA version to 6 with my latest commit and fixed the icon.

**Review by ibot3 — CHANGES_REQUESTED** [review]
Needs some improvement ;)
Better test it yourself.

File: sipa/templates/base.html (hunk 1 above)
Comment (FestplattenSchnitzel) [reply sipa/templates/base.html]: Done.

**Pull request merged**